SPECS/postgresql/postgresql.spec

```diff
@@ -18,6 +18,8 @@ BuildRequires:  perl
 BuildRequires:  readline-devel
 BuildRequires:  tzdata
 BuildRequires:  zlib-devel
+BuildRequires:  pkgconfig(icu-i18n)
+BuildRequires:  pkgconfig(icu-uc)
```

Comment:
Could I ask you to put the new BRs in alphabetical order, since it's already neatly kept that way?

Reply:
Done now